mne_bids/tests/test_utils.py

```diff
@@ -113,6 +113,16 @@ def test_age_on_date():
         _age_on_date(bday, exp4)
 
 
+def test_get_brainvision_encoding():
```

Comment:
could you explain a bit more? - its the first test I write :)

Reply:
I am at a loss as well :-D +1 for an explanation @jasmainak - which regression? We just had a bug report and now we have a new feature covering that bug. The test verifies that the feature works as expected.

Reply:
since you had a bug which we are trying to fix, the first step would be to reproduce the bug in a test. Otherwise, in the future, if someone changes something that reintroduces the bug, the test will not fail. Basically, I am asking -- can you test the corner case that you hit instead of testing for UTF-8 ?

Reply:
but I see your point. Ill check what I can do (python 3 apparently doesnt want users to change global decoding)